### PR TITLE
Clickable names in checker statistic view

### LINKED ISSUE #905: Clickable names and severity levels in checker statistic view
We should be able to click on the names of the checkers in checker statistics view to see all the results from a check. 

If a severity level is selected a filter view would be shown with the results filtered to that level.

### PR DESCRIPTION
**Pull request by csordasmarton**
Closes #905

### CODE REVIEW

File: www/scripts/codecheckerviewer/CheckerStatistics.js [hunk 1]
```diff
@@ -208,4 +298,4 @@ function (declare, ItemFileWriteStore, Deferred, all, Memory, Observable,
       this.addChild(this._checkerStatistics);
     }
   });
-});
+});
```
Comment (Xazax-hun) [inline www/scripts/codecheckerviewer/CheckerStatistics.js]: I think this change is unneeded and usually, it is good to preserve the last end of line in files.

File: www/scripts/codecheckerviewer/CheckerStatistics.js [hunk 2]
```diff
@@ -129,6 +140,81 @@ function (declare, ItemFileWriteStore, Deferred, all, Memory, Observable,
...
+        case 'falsePositive':
+          if (!item.falsePositive[0]) return;
+
+          state[reviewStatusFilter.class] =reviewStatusFilter.stateConverter(
+            CC_OBJECTS.ReviewStatus.FALSE_POSITIVE);
+          break;
+        case 'intentional':
+          if (!item.intentional[0]) return;
+
+          state[reviewStatusFilter.class] =reviewStatusFilter.stateConverter(
```
Comment (Xazax-hun) [inline www/scripts/codecheckerviewer/CheckerStatistics.js]: Is this formatting conventional for javascript? (No space after assignment)

Comment (csordasmarton) [reply www/scripts/codecheckerviewer/CheckerStatistics.js]: No, this is definitely not. Thanks.

**Comment by csordasmarton** [comment]
Yes, I know about this problem. It will be fixed at #620 issue after I replaced the run id with the run name in permalinks. I think I will do this today.

**Pull request merged**